# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1534263927 25200
#      Tue Aug 14 09:25:27 2018 -0700
# Node ID 73a1eb82c3a2708321ded98976508d8d0138151a
# Parent  842b6729d3ec76b6a09756002ee3b3ce1c4748f7
doc: remove isfile and isfolder from the missing functions list

* __unimplemented__.m: Remove isfile and isfolder from the list.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -900,18 +900,16 @@ function rlist = missing_functions ()
   "isConnected",
   "isdag",
   "isdatetime",
   "isdst",
   "isduration",
   "isEdge",
   "isenum",
   "isevent",
-  "isfile",
-  "isfolder",
   "ishole",
   "isIllConditioned",
   "isinterface",
   "isinterior",
   "isInterior",
   "isisomorphic",
   "islocalmax",
   "islocalmin",
